cluster/charts/crossplane/crds/pkg.crossplane.io_controllerconfigs.yaml

```diff
@@ -553,6 +553,18 @@ spec:
                         description: User is a SELinux user label that applies to the container.
                         type: string
                     type: object
+                  seccompProfile:
```

Comment:
Note for others: this is a non-breaking API change to `ControllerConfig`. Seccomp was promoted to security context / pod security context from annotations upstream, which is now reflected here 👍